src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -75,6 +79,11 @@ public static partial class JNIEnv {
 			}
 		}
 
+		internal static BoundExceptionType BoundExceptionType {
```

Comment:
It's internal; just leave it as a field, so we don't have to rely on the inliner?  This shouldn't even be used from outside this class, anyway...